src/library/scala/collection/mutable/ArrayBuffer.scala

```diff
@@ -54,6 +53,7 @@ class ArrayBuffer[A] private (initialElements: Array[AnyRef], initialSize: Int)
 
   @transient private[this] var mutationCount: Int = 0
 
+  // needs to be private[collection] for parallel-collections
```

Comment:
oops, this is out of sync with the code (`private` vs `protected`)

Reply:
@SethTisue that was actually intentional. `protected[x]` is the union of `protected` and `private[x]`. to fulfill the needs of the stdlib (`PriorityQueue` in particular), it must be `protected`. parallel-collections requires it to be `private[collection]`, but does not make use of it being `protected`.

Reply:
Okay — but the comment doesn't communicate that to the reader. As written, it _reads_ like a mistake.

Reply:
... stop making good points Seth